frontend/src/pages/Dashboard/data/reducer.ts

```diff
@@ -52,6 +52,16 @@ const dashboardReducer = (
         ...state,
         error: action.payload.error,
       };
+    case ActionTypes.CHANGE_PROFILE_ICON_SUCCESS:
+      return {
+        ...state,
+        user: state.user
```

Comment:
I don't think you can get here without a user object, so it should be safe to just spread the user (unless typescript does not enjoy it?)

Reply:
ya i tried doing user: { ...state.user } it gives me an error lol so that's why i added a check there if that's what u meant

Reply:
I think it might make sense to do a `//@ts-ignore` since I think it would make the code much more readable. Up to you tho

Reply:
okie i think ima keep it this way just cuz if state.user is undefined i dont want it to blow up hehe